tests/python/unittest/test_tir_usmp_algo_hill_climb.py

```diff
@@ -22,7 +22,7 @@
 from tvm.tir.usmp.utils import BufferInfo, PoolInfo
 
 
-def _check_max_workspace_size(buffer_pool_allocations, pool_info, size):
+def _check_max_workspace_size(buffer_pool_allocations, pool_info, size, tolerance=0):
```

Comment:
Is there a test where tolerance is not zero ?

Reply:
def test_intervals(intervals)  accepts 5% of difference now